civil/civil_test.go

```diff
@@ -163,6 +163,29 @@ func TestDateArithmetic(t *testing.T) {
 	}
 }
 
+// Several ways of getting from Fri Nov 18 2011 to Thu Mar 19 2016
```

Comment:
The unit tests are all copied from https://cs.opensource.google/go/go/+/refs/tags/go1.20.1:src/time/time_test.go.

Reply:
Cool that's a great idea.